applications/nrf5340_audio/src/modules/audio_usb.c

```diff
@@ -42,29 +45,29 @@ static void data_write(const struct device *dev)
 
 	void *data_out;
 	size_t data_out_size;
-	struct net_buf *buf_out;
+	struct net_buf *audio_buf_out;
 
-	buf_out = net_buf_alloc(&pool_out, K_NO_WAIT);
+	audio_buf_out = net_buf_alloc(&pool_out, K_NO_WAIT);
 
 	ret = data_fifo_pointer_last_filled_get(fifo_tx, &data_out, &data_out_size, K_NO_WAIT);
 	if (ret) {
 		tx_num_underruns++;
 		if ((tx_num_underruns % 100) == 1) {
 			LOG_WRN("USB TX underrun. Num: %d", tx_num_underruns);
 		}
-		net_buf_unref(buf_out);
+		net_buf_unref(audio_buf_out);
 
 		return;
 	}
 
-	memcpy(buf_out->data, data_out, data_out_size);
+	memcpy(audio_buf_out->data, data_out, data_out_size);
 	data_fifo_block_free(fifo_tx, data_out);
 
 	if (data_out_size == usb_audio_get_in_frame_size(dev)) {
-		ret = usb_audio_send(dev, buf_out, data_out_size);
+		ret = usb_audio_send(dev, audio_buf_out, data_out_size);
 		if (ret) {
 			LOG_WRN("USB TX failed, ret: %d", ret);
-			net_buf_unref(buf_out);
+			net_buf_unref(audio_buf_out);
```

Comment:
Missing a return here? 

Else if `!tx_first_data` then you also log `LOG_INF("USB TX first data sent.");` after `LOG_WRN("USB TX failed, ret: %d", ret);`

Reply:
Yeah, returned added